net-vpn/strongswan: fix build failure when stroke plugin is enabled

kernlpanic opened the pull request:
Closes: https://bugs.gentoo.org/957173

When using two plugins together, vici and strongswan, the build fails if non-root is enabled. This pull request should fix the issue.

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

thesamesam commented:
> When using two plugins together, vici and strongswan, the build fails if non-root is enabled. 

Please always include remarks like this in the commit message too.

kernlpanic commented:
> > When using two plugins together, vici and strongswan, the build fails if non-root is enabled.
> 
> Please always include remarks like this in the commit message too.

I updated the commit message. Is it better like this or should I put into into the commit description? I also updated the pull request description and the pull request title because it was wrong, but I tested different permutations of the use flags to be sure everything works

kernlpanic commented:
> > I updated the commit message. Is it better like this or should I put into into the commit description? I also updated the pull request description and the pull request title because it was wrong, but I tested different permutations of the use flags to be sure everything works
> 
> I would have the summary (first line/title) be a little bit shorter, and then have the stuff underneath (description) be longer.
> 
> Thanks for asking and caring about that :)

You're welcome. Thank you for taking the time to teach me the correct way.
I changed the commit message to a shorter version now and added the additional information into the commit description.
